macos_environment.sh

```diff
@@ -16,8 +16,9 @@ if [ -z "$INPUTLEAP_BUILD_ENV" ]; then
 
     printf "Modifying environment for InputLeap build...\n"
 
-    if command -v port; then
-        printf "Detected Macports\n"
+    # Check if user preference is set for MacPorts
+    if [ "$LEAP__SYS_PKGMGR" = "macports" ]; then
```

Comment:
If you check both `LEAP__SYS_PKGMGR` and `command -v port` then this if could handle both cases

Reply:
i.e. if LEAP__SYS_PKGMGR == macports OR command -v port.

Reply:
Also I think it's better to reuse INPUTLEAP_BUILD_MACPORTS and  INPUTLEAP_BUILD_BREW instead of introducing LEAP__SYS_PKGMGR.